digitalstrategie/assets/scss/_variables.scss

```diff
@@ -171,7 +171,7 @@ $transition-collapse-width:   width .35s ease !default;
 $font-family-sans-serif:      "Arial", "Helvetica", sans-serif !default;
 $font-family-monospace:       SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
 // stylelint-enable value-keyword-case
-$font-family-base:            var(--#{$variable-prefix}font-sans-serif) !default;
+$font-family-base:            "BerlinType" !default;
```

Comment:
I would add the type to `$font-family-sans-serif` as thats used in more places by BS then base, I would also add a backup font stack (like above) incase there are issues with font loading in older browsers